README.md

```diff
@@ -15,3 +15,20 @@ For citing PyFMI, please use:
 Andersson, C, Åkesson, J & Führer, C 2016, PyFMI: A Python Package for Simulation of Coupled Dynamic Models with the Functional Mock-up Interface. Technical Report in Mathematical Sciences, nr. 2, vol. 2016, vol. LUTFNA-5008-2016, Centre for Mathematical Sciences, Lund University.
 
 For information about contributing, see [here](https://github.com/modelon/contributing).
+
+Installation from source
+-----------
+PyFMI can be built and installed from source, note this requires [FMI library](https://github.com/modelon-community/fmi-library). Additional details are found in the file `setup.py`.
```

Comment:
also Assimulo and dependencies within?

Maybe worthwhile to update Assimulo Readme in the same turn.

Reply:
Will do!